cmd/environment_to_ini_test.go

```diff
@@ -0,0 +1,64 @@
+// Copyright 2019 The Gitea Authors. All rights reserved.
+// Use of this source code is governed by a MIT-style
+// license that can be found in the LICENSE file.
+
+package cmd
+
+import "testing"
+
+func TestDecodeSectionKey(t *testing.T) {
+	tests := []struct {
+		name    string
+		encoded string
+		section string
+		key     string
+	}{
+		{
+			name:    "Simple",
+			encoded: "Section__Key",
```

Comment:
Why encoded value are not prepended with `EnvironmentPrefix` or hardcoded "GITEA__" ?
Looking at https://github.com/go-gitea/gitea/pull/7287/files#diff-15c1c17f9bcede7466e3befd8d1ac01bR55 they are not stripped before being sent to DecodeSectionKey so this test doesn't seems to really represent actual value presented.

Reply:
Oh damn!

Have removed the prefix before passing to DecodeSectionKey instead of changing the DecodeSectionKey function.

Sorry it's taken me so long.